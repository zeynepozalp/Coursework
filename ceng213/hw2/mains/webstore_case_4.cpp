#include <iostream>
#include <fstream>
#include <sstream>
#include <cstdlib>

#include "stock_photo_webstore.hpp"
#include "photo.hpp"

int main() {
    StockPhotoWebstore webstore;

    webstore.addPhoto(Photo("music", "guitar", "res_low", 15))
            .addPhoto(Photo("music", "piano", "res_medium", 20))
            .addPhoto(Photo("music", "bass", "res_high", 25))
            .addPhoto(Photo("music", "drum", "res_low", 15))
            .addPhoto(Photo("music", "harp", "res_medium", 20))
            .addPhoto(Photo("food", "fish", "res_high", 20))
            .addPhoto(Photo("food", "chips", "res_low", 10))
            .addPhoto(Photo("food", "tacos", "res_medium", 15))
            .addPhoto(Photo("food", "nachos", "res_high", 20))
            .addPhoto(Photo("sports", "golf", "res_low", 30));

    webstore.removePhoto("music", "bass")
            .removePhoto("food", "chips")
            .removePhoto("sports", "golf")
            .printAllPhotos();

    return 0;
}
