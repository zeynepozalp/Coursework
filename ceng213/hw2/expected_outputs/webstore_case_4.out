{"food" : {"fish" : "food;fish;res_high;20", "nachos" : "food;nachos;res_high;20", "tacos" : "food;tacos;res_medium;15"}, "music" : {"drum" : "music;drum;res_low;15", "guitar" : "music;guitar;res_low;15", "harp" : "music;harp;res_medium;20", "piano" : "music;piano;res_medium;20"}, "sports" : {}}
